storage/src/main/java/tech/pegasys/teku/storage/server/Database.java

```diff
@@ -105,6 +105,8 @@ public interface Database extends AutoCloseable {
   @MustBeClosed
   Stream<SignedBeaconBlock> streamHotBlocks();
 
+  long countFinalizedBlocks();
```

Comment:
Added this because the count using the old stream was very slow... I was running it at the start of migration, which I think is probably something to keep, because it lets you track percent complete...  Possibly it'd be worth naming it `countFinalizedUnblindedBlocks`, as it's actually checking old storage columns...